packages/dds/sequence/src/revertibles.ts

```diff
@@ -16,6 +16,8 @@ const IntervalEventType = {
 	PROPERTYCHANGED: 3,
 } as const;
 
+export const idMap = new Map<string, string>();
```

Comment:
Don't export this. The important thing is to test that the outward behaviour is correct where revertibles function correctly even when an interval has been deleted and undone. E.g. move interval, delete interval, undo delete, undo move. (For which you still need to read the map in the revert functions.)